src/test/java/edu/harvard/iq/dataverse/dataverse/validation/BannerErrorHandlerTest.java

```diff
@@ -0,0 +1,152 @@
+package edu.harvard.iq.dataverse.dataverse.validation;
+
+import com.google.common.collect.Sets;
+import edu.harvard.iq.dataverse.dataverse.banners.BannerLimits;
+import edu.harvard.iq.dataverse.dataverse.banners.DataverseBanner;
+import edu.harvard.iq.dataverse.dataverse.banners.DataverseLocalizedBanner;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.omnifaces.util.Faces;
+
+import javax.faces.application.FacesMessage;
+import javax.faces.context.ExternalContext;
+import javax.faces.context.FacesContext;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Locale;
+
+import static edu.harvard.iq.dataverse.util.DateUtil.convertToDate;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BannerErrorHandlerTest {
+
+    @Mock
+    private FacesContext facesContextMock;
+    @Mock
+    private ExternalContext externalContextMock;
+    @Mock
+    private Locale locale;
```

Comment:
It's easier to just create locale by hand than mocking it `Locale.ENGLISH`

Reply:
I tried that, but after compilation it still screams that it needs `Locale` even though `Locale.ENGLISH` returns it, it still doesn't quite like it.